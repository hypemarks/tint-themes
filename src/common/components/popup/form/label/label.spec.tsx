import { render } from '@testing-library/react';
import React from 'react';
import Label from './label';

describe('Test <Label /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(<Label id={'id'} label={'label'} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
