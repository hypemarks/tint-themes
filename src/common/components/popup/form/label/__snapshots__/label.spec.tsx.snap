// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Label /> Component should render snapshot 1`] = `
<DocumentFragment>
  <label
    class="undefined"
    for="id"
  >
    label
  </label>
</DocumentFragment>
`;
